internal/http/services/owncloud/ocdav/propfind.go

```diff
@@ -63,10 +65,12 @@ const (
 
 // ns is the namespace that is prefixed to the path in the cs3 namespace
 func (s *svc) handlePathPropfind(w http.ResponseWriter, r *http.Request, ns string) {
-	ctx := r.Context()
-	ctx, span := trace.StartSpan(ctx, "propfind")
+	ctx := rtrace.Propagator.Extract(r.Context(), propagation.HeaderCarrier(r.Header))
```

Comment:
isn't this done by https://github.com/cs3org/reva/pull/1984/files#diff-e970327f0efe1f62011a0842762a6c652f53979a5c0fcec42b72897e798ac646R305 already?

Reply:
yes 🙈